### PR TITLE
1.0 release changes

### PR DESCRIPTION
**Pull request by natebird**
- Update dependencies to support Fluent 1.0.x and PostgreSQL 1.0.x
- Updates to support Node functionality
- Add many more tests similar to the MySQL and SQLite drivers
- Added .gitignore
- Updated README
- Code and documentation cleanup
- Add Travis and Circle configuration files
- Override .data type to be "BYTEA"

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PackageDescription
 let package = Package(
     name: "FluentPostgreSQL",
     dependencies: [
-        .Package(url: "https://github.com/qutheory/postgresql.git", majorVersion: 0, minor: 1),
-        .Package(url: "https://github.com/qutheory/fluent.git", majorVersion: 0, minor: 7)
+        .Package(url: "https://github.com/vapor/postgresql.git", majorVersion: 1, minor: 0),
```
Comment (tanner0101) [inline Package.swift]: Internal vapor components should rely on `majorVersion`s only.

File: Sources/FluentPostgreSQL/PostgreSQLDriver.swift [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import Fluent
+import PostgreSQL
+
+public class PostgreSQLDriver: Fluent.Driver {
+    public var idKey: String = "id"
+    public var database: PostgreSQL.Database
+
+    /**
+     Attempts to establish a connection to a PostgreSQL database engine.
```
Comment (tanner0101) [inline Sources/FluentPostgreSQL/PostgreSQLDriver.swift]: Comment formatting here is a bit off.

```
/**
    foo
*/
func bar()
```

File: Package.swift (hunk 1 above)
Comment (natebird) [reply Package.swift]: Ok. Good to know.

File: Sources/FluentPostgreSQL/PostgreSQLDriver.swift (hunk 2 above)
Comment (natebird) [reply Sources/FluentPostgreSQL/PostgreSQLDriver.swift]: Ok. I'll fix all of them.

File: Package.swift (hunk 1 above)
Comment (natebird) [reply Package.swift]: Does that apply to just postgresql or to fluent too?

Comment (natebird) [reply Package.swift]: Updated.

**Comment by natebird** [comment]
@tannernelson Anything else need to be done here?

**Review by tanner0101 — APPROVED** [review]

**Pull request merged**